# Publications

OrcID: [0009-0007-3250-5299](https://orcid.org/0009-0007-3250-5299)

[Google Scholar](https://scholar.google.com/citations?view_op=list_works&hl=en&user=O533DuIAAAAJ)

## G-quadruplexes self-assembled from nucleotide monomers as stable prepolymer scaffolds in aqueous environments

Scientific Reports | 2026-02-07

??? abstract "Abstract:"

    
    Life is composed of genetic and functional polymers, such as nucleic acids. For life to have emerged, a prebiotic mechanism for assembling these polymers is essential. In a prebiotic environment teeming with diverse organic molecules, selecting, concentrating and bringing together only relevant building blocks poses a significant challenge. G-quadruplexes, a secondary structure of DNA and RNA, are known to self-assemble from nucleotide monomers, creating an ideal preassembly for nucleotide polymerization. We investigate the detailed structure of self-assembled G-quadruplexes using high-resolution atomic force microscopy (AFM) measurements in solution. We show that G-quadruplexes of nucleotide monomers are stable on surfaces in aqueous solution at concentrations orders of magnitude below their solubility limit. When subjected to cycles of evaporation and rehydration at elevated temperatures, the G-quadruplexes partially transform into extended, RNA-like structures, also stable on surfaces in solution, consistent with a polymeric nature. G-quadruplexes self-assembled from nucleotide monomers could have served as persistent prepolymer scaffolds, providing genuine molecular selectivity in prebiotic environments.

[Article](https://www.nature.com/articles/s41598-026-38899-5)

## Classical and machine learning algorithms for analysing complex DNA structures (Thesis)

White Rose eTheses Online | Embargoed until: 18 July 2026

??? abstract "Abstract:"

    Atomic force microscopy (AFM) is unique in its ability to image single molecules in liquid with sub-molecular resolution, without the need for labelling or averaging. This enables us to probe biomolecular structures in native-like states and examine conformational changes. For DNA, its innate flexibility enables compaction in the nucleus and processing by essential cellular machinery which drives a large range of these conformational changes and must be regulated to ensure cell survival. However, the large quantities of closed AFM filetypes limit the adoption of open-source tools developed by the image analysis community. The lack of AFM-specific automated analysis tools to process raw data and characterise conformation make high-throughput conformational analyses difficult and laborious.

    I have developed AFMReader, an open-source Python file loader for the extraction of AFM images and metadata from proprietary file formats. I also developed TopoStats, a toolbox for; AFM-specific image processing, object identification, and characterisation of individual molecules. Key developments to this pipeline are a new height-biased skeletonisation algorithm, and quantification of overlapping DNA segments, enabling the accurate tracing of branched, crossing, and overlapping DNA structures.

    This new automated tracing architecture enables the classification of DNA knots and catenanes produced by the Xer recombination system by extracting a pseudo 3D molecular backbone trace. I characterise DNA replication fork stalling by Lac-repressor protein and the Tur-Ter complex via calculation of the replicated and unreplicated DNA segment contour lengths. I show that this pipeline can be adapted to characterise a possible prebiotic RNA synthesis pathway via molecular backbone height profiles across samples conditions in a fixed location. Finally, I explore the feasibility of a deep learning variational auto-encoder to describe the conformational landscape of supercoiled DNA minicircles. These applications show the versatility of this new pipeline as a toolbox to help quantify and uncover the role of structure in DNA interactions.

[Thesis](https://etheses.whiterose.ac.uk/id/eprint/37187/)

## Quantifying complexity in DNA structures with high resolution Atomic Force Microscopy

Nature Communications | 2025-07-01

??? abstract "Abstract:"

    DNA topology is essential for regulating cellular processes and maintaining genome stability, yet it is challenging to quantify due to the size and complexity of topologically constrained DNA molecules. By combining high-resolution Atomic Force Microscopy (AFM) with a new high-throughput automated pipeline, we can quantify the length, conformation, and topology of individual complex DNA molecules with sub-molecular resolution. Our pipeline uses deep-learning methods to trace the backbone of individual DNA molecules and identify crossing points, efficiently determining which segment passes over which. We use this pipeline to determine the structure of stalled replication intermediates from Xenopus egg extracts, including theta structures and late replication products, and the topology of plasmids, knots and catenanes from the E. coli Xer recombination system. We use coarse-grained simulations to quantify the effect of surface immobilisation on twist-writhe partitioning. Our pipeline opens avenues for understanding how fundamental biological processes are regulated by DNA topology.

[Article](https://www.nature.com/articles/s41467-025-60559-x)

## Open Research Case Study: Alice Pyne and the TopoStats team-Developing open source software aligned with the FAIR4RS principles: The TopoStats project

Sheffield University | 2023-07-19

??? abstract "Abstract:"

    Case study detailing the open research practices for which Dr Pyne and the TopoStats team received the University of Sheffield Open Research Prize 2023 (team category).  This prize was awarded for the team's work on TopoStats, an open source software tool in the Atomic Force Microscopy (AFM) field which adheres to the FAIR4RS (FAIR for Research Software) principles.

[Article](https://orda.shef.ac.uk/articles/report/Open_Research_Case_Study_Alice_Pyne_and_the_TopoStats_team_-_Developing_open_source_software_aligned_with_the_FAIR4RS_principles_The_TopoStats_project/23702109/1/files/41596875.pdf)